packages/sdk/src/index.ts

```diff
@@ -606,6 +606,14 @@ export {
   uniswapV2LiquidityRedeemEncoding,
   type UniswapV2LiquidityRedeemArgs,
 } from "./extensions/integrations/instances/uniswapV2Liquidity.js";
+// ./extensions/integrations/instances/uniswapV2Liquidity.js
+
+export {
+  encodeUniswapV3TakeOrderArgs,
+  decodeUniswapV3TakeOrderArgs,
+  uniswapV3TakeOrderEncoding,
+  type UniswapV3TakeOrderArgs,
+} from "./extensions/integrations/instances/uniswapV3.js";
```

Comment:
Make sure you generate these exports with the script.

Reply:
done